### ISSUE TITLE
improper splitting of certain imports

### ISSUE THREAD

**Issue by davidbielik** [0DarkPhoenix/split-mui-imports#1]
These two are both being incorrectly refactored, when they only exist as a named export

```
import { styled } from '@mui/material';
import { Theme, createTheme } from '@mui/material';
```

Is there a way to configure exclusions, or some comment to ignore lines? (i.e. `// mui-split-ignore`)

Thanks!

**Comment by 0DarkPhoenix** [comment]
Thank you very much for reporting this issue!

Currently, there is no way to add certain MUI imports to some type of ignore list or add a comment to ignore the import. This is a really good suggestion, and I'll put some extra time into finding out which special cases an import shouldn't be split (Like this situation where { Theme, createTheme } and { styled } should stay like this).

**Comment by 0DarkPhoenix** [comment]
I've just released 0.3.0, which added the following way to manage which imports and modules are allowed to be split:

Allowed Imports: Manage a list of import paths which should be split
Excluded Modules: Manage modules which shouldn't be split when they are imported from an allowed import

![image](https://github.com/user-attachments/assets/b40588fe-1e56-4ec3-90a0-74418b0beedd)

This is what appears in settings.json when you click on "Edit in settings.json" for the allowed imports and excluded modules:
```       
 "splitMuiImports.allowedImports": [
        "@mui/icons-material",
        "@mui/material",
        "@mui/lab"
    ],
    "splitMuiImports.excludedModules": [
        "adaptV4Theme",
        "createColorScheme",
        "createStyles",
        "createTheme",
        "Experimental_CssVarsProvider",
        "experimental_sx",
        "experimentalStyled",
        "getOverlayAlpha",
        "makeStyles",
        "responsiveFontSizes",
        "shouldSkipGeneratingVar",
        "styled",
        "THEME_ID",
        "Theme",
        "ThemeProvider",
        "unstable_createMuiStrictModeTheme",
        "useTheme",
        "useThemeProps",
        "withStyles",
        "withTheme"
    ],
```
